README.md

```diff
@@ -1,116 +1,57 @@
-## Grumbler Scripts
+## Grumbler Scripts Monorepo
 
-```
-npm install @krakenjs/grumbler-scripts
-```
+This is a collection of configuration scripts one can use for their projects.
 
-Shared scripts for grumbler based modules.
+### Packages Available
 
-## Package.json scripts
+Below is a list of available packages to install. To provide a one stop shop we will continue to support `@krakenjs/grumbler-scripts` which is a combination of all the packages. Feel free to use that parent package or install configs as you need.
```

Comment:
Great idea with still providing a single global package for ease of use while still allowing folks to pick and choose the features they want ❤️